File: 2486 append-characters-to-string-to-make-subsequence/solution.java
class Solution {
    public int appendCharacters(String s, String t) {
        int j=0,ct=0;
        for(char c:s.toCharArray()){
            if(j<t.length() && c==t.charAt(j)) {
                ct++;
                j++;
            }
        }
        return t.length()-ct;
    }
}

File: 2486 append-characters-to-string-to-make-subsequence/solution.cs
public class Solution {
    public int AppendCharacters(string s, string t) {
        int j=0,ct=0;
        foreach(var c in s) {
            if(j<t.Length && c==t[j]){
                ct++;
                j++;
            }
        }
        return t.Length-ct;
    }
}
